File: LIB/tempoci.f90
MODULE tempoci
IMPLICIT NONE

REAL :: primoOH,secoOH,primosci,ycarbo
INTEGER,PARAMETER :: mxsci=15
INTEGER :: nnsci
REAL :: yysci(mxsci)
CHARACTER(LEN=120) :: chemsci(mxsci)

INTEGER,PARAMETER :: mxcarb=6
CHARACTER(LEN=120):: focarb(mxcarb)
INTEGER           :: nc_carb(mxcarb)
REAL              :: ycarb(mxcarb)

INTEGER :: c1ciflag

END MODULE tempoci

File: LIB/tempoci.c
/* Temporary OCI data module */
#include "tempoci.h"

double primoOH, secoOH, primosci, ycarbo;

int nnsci;
double yysci[MXSCI];
char chemsci[MXSCI][120];

char focarb[MXCARB][120];
int nc_carb[MXCARB];
double ycarb[MXCARB];

int c1ciflag;
